features/DVO_Recommendations/DVO_recommendations.feature

```diff
@@ -46,7 +54,7 @@ Feature: Current way of presenting workload related recommendations is not good
           | Recommendations     | no                         |
           | Workloads           | yes                        |
      When user select "Workloads" menu item from this sub-menu
-     Then an "Workloads" page should be displayed right of the left menu bar
+     Then a "Workloads" page should be displayed right of the left menu bar
```

Comment:
"Advisor Workload" page, based on the UX design

Reply:
comment applies on line 99, 163, and 234 too, if applied

Reply:
fixed, thank you